src/styles/index.scss

```diff
@@ -1,3 +1,255 @@
+$stars-size: 16px;
+
 body {
   margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
+  font-weight: 500;
+}
+
+.container {
+  --column-count: 4;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 46px 48px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
+
+/* ====== Header ====== */
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding: 3px 0 0;
+}
+
+.menu {
+  font-size: 12px;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.link {
+  position: relative;
+  margin: 0 10px;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  display: flex;
+}
+
+.menu li:first-child a {
+  margin-left: 0;
+}
+
+.menu li:last-child a {
+  margin-right: 0;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  margin: 0 10px;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  display: flex;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 2px;
+  transform-origin: bottom left;
+}
+
+.link:hover::after {
+  background-color: rgba(0, 172, 220, 1);
+}
+
+.container_logo {
+  width: 40px;
+}
+
+.free {
+  flex-grow: 1;
+}
+
+/* ====== Product cards ====== */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: $stars-size;
+  height: $stars-size;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  justify-content: center;
+  flex-direction: column;
+}
```

Comment:
The `.card` class seems to be a block element, yet it has flex properties such as `justify-content` and `flex-direction` without displaying as flex. If these properties are not needed, they should be removed to avoid confusion.